auto_submit/lib/service/validation_service.dart

```diff
@@ -477,36 +481,25 @@ class ProcessMergeResult {
 typedef RetryHandler = Function();
 
 /// Internal wrapper for the logic of merging a pull request into github.
-Future<graphql.QueryResult> _processMergeInternal({
-  required graphql.GraphQLClient client,
-  required Config config,
-  required QueryResult queryResult,
-  required github.PullRequest messagePullRequest,
+Future<github.PullRequestMerge> _processMergeInternal(
+  GithubService githubService,
```

Comment:
For larger method signatures, it's easier to use if everything is a named parameter.

Generally, whenever there's 2+ variables I make everything named.

Reply:
Done!